fusillade/api/__init__.py

```diff
@@ -0,0 +1,204 @@
+import collections
+import functools
+import logging
+import os
+import re
+
+import chalice
+import requests
+
+from connexion import FlaskApp
+from connexion.decorators.validation import ParameterValidator, RequestBodyValidator
+from connexion.resolver import RestyResolver
+
+from fusillade import FusilladeBindingException, Config
+
+
+class FusilladeParameterValidator(ParameterValidator):
```

Comment:
`FusilladeRequestBodyValidator` and `FusilladeParameterValidator` have fairly similar code.  Is there a way to consolidate any of it?

Reply:
how would you consolidate?